packages/markdown-processor/src/server/index.test.ts

```diff
@@ -1,5 +1,10 @@
+import { h } from "hastscript";
+import { visit } from "unist-util-visit";
+
 import { parseMarkdownToHTML } from ".";
 
+import type { Element, Node } from "hast";
```

Comment:
https://github.com/saitamau-maximum/markdown/pull/12#discussion_r2292695552


```suggestion
import { h } from "hastscript";
import { visit } from "unist-util-visit";
import type { Element, Node } from "hast";

 import { parseMarkdownToHTML } from ".";
```

Reply:
いや分からんてw
import sort orderのことね

Reply:
type importを後回しにするようになってて、 eslint configを変えないとautofixでこの形になるね
別PRで直します